src/main/java/com/softserve/entity/Room.java

```diff
@@ -43,6 +42,6 @@ public class Room implements Serializable {
     @Column(name = "disable", columnDefinition = "boolean default 'false'")
     private boolean disable = false;
 
-    @Column(name = "sort_order")
-    private Double sortOrder;
+    @Column(name = "sorting_order")
```

Comment:
why change the column name?
https://trends.google.ru/trends/explore?date=all&q=sort%20order,sorting%20order

https://books.google.com/ngrams/graph?content=sort+order%2Csorting+order&year_start=1800&year_end=2019&corpus=26&smoothing=3&case_insensitive=true